contracts/Account.cairo

```diff
@@ -35,6 +35,10 @@ end
 func public_key() -> (res: felt):
 end
 
+@storage_var
+func _is_account() -> (res: felt):
+end
+
```

Comment:
```suggestion
```
I don't think this is needed at all. `is_account` can just return `1`

Reply:
Also consider [Julien's comment regarding returning a magic value](https://github.com/OpenZeppelin/cairo-contracts/discussions/100#discussioncomment-1969470).